sleap_nn/training/model_trainer.py

```
@@ -209,6 +213,26 @@ def __init__(
                 "symmetries": symm,
             }
 
+        # if edges and part names aren't set in config, get it from `sio.Labels` object.
+        head_config = self.config.model_config.head_configs[self.model_type]
+        for key in head_config:
+            if "part_names" in head_config[key].keys():
+                if head_config[key]["part_names"] is None:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)